docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/README.md

```diff
@@ -8,29 +8,52 @@ keywords: file storages, Nextcloud error, Nextcloud troubleshooting, OneDrive er
+| ERR_CLIENT_INVALID     | The configured client credentials are invalid.                                        | Either the client id or the client secret are invalid. The error descriptions should help you finding the culprit.                                                                                        | Go to the correct application overview in the [Azure Portal](https://portal.azure.com/). Copy the correct client id to the input field, or check whether the client secret is still valid. Attention: secrets might have a expiration date. If a secret is expired, you'll have to generate a new one.                                                                         |
+| ERR_DRIVE_INVALID      | The configured drive cannot be found.                                                 | The request for the drive id failed with not finding the drive id. The drive might be deleted, or your application has no permissions to see it.                                                          | Consult the [drive guide](../../../integrations/one-drive/drive-guide/) and fetch the desired drive id again, to fill out the input field.                                                                                                                                                                                                                                     |
+| WRN_UNEXPECTED_CONTENT | The connection request was successful, but unexpected content was found in the drive. | This warning is only shown, if the file storage is configured to automatically manage project folder permissions. There was data found in the drive, that is not a project folder created by OpenProject. | Go to your drive and migrate or delete the data from the drive root, that was not created by OpenProject. A drive configured for usage with the *Automatically managed project folders* option has a disrupted inheritance chain. Any data in here can only be seen by site owner. It is discouraged to use this drive for other purposes as with the OpenProject integration. |
+| ERR_UNKNOWN            | An unknown error occurred.                                                            | There can be multiple reasons and the error source was not foreseen.                                                                                                                                      | Errors of this kind are logged to the server logs. Look for a log entry starting with `Connection validation failed with unknown error:`                                                                                                                                                                                                                                       |
+
+#### Nextcloud compatibility
```

Comment:
@Kharonus 
I am wondering about the reason of having a own section for "Nextcloud compatibility"?

Reply:
As just clarified, the heading might be misleading. I tried to have here a placeholder, that explains to the reader, that this is not available for nc file storages (yet). Once the feature is done, this section will be deleted, and the rest of the text updated.